.github/workflows/build.yml

```diff
@@ -3,6 +3,14 @@ name: Build
 on:
   workflow_dispatch:
   pull_request:
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - '.github/config.json'
+      - 'bin/**'
+      - '.gitignore'
+      - 'composer.lock'
```

Comment:
I would remove this as it affects the build (apply this comment to other actions too)